internal/driver/mobile/mobileinit/mobileinit_ios.m

```diff
@@ -1,5 +1,5 @@
-// +build darwin
-// +build arm arm64
+//go:build ios
```

Comment:
This is using newer Go build tags - not supported all the way back to 1.14

Reply:
Hmm. Good point, my editor did auto-fixes for these tags. Is there a particular reason we have to support 1.14 builds? Especially Go 1.14 is not officially supported by the Go team anymore.

Furthermore, 1.14 and 1.16 have a significant difference that 1.14 ios are building via // +build darwin/arm64, whereas 1.16 builds ios via // +build ios.

Reply:
Yes, we have to support Go 1.14. It is the newest version available on some OS and the last to support building iOS 32-bit

Reply:
The old format of build flags is still supported by Go 1.17 so there is no need to move away from the old type.

Reply:
The formatting in `gofmt` should (by default with older versions in go.mod, I think) add both without breaking for older Go versions. I think it could be a stop gap, but might be better to do that all in a separate PR to not add complexity here.

Reply:
I can send a prototype to show a global change on the build tags :)